SerializationTest fails with Vaadin 25.1 due to missing jakarta.servlet-api

\`SerializationTest\` fails when building against Vaadin 25.1.0-rc2 with the v25 profile.

**Error:**
```
java.lang.NoClassDefFoundError: jakarta/servlet/http/HttpSessionBindingListener
    at com.flowingcode.vaadin.addons.extendedlogin.test.SerializationTest.testSerializationOf(SerializationTest.java:37)
    at com.flowingcode.vaadin.addons.extendedlogin.test.SerializationTest.testSerialization(SerializationTest.java:48)
Caused by: java.lang.ClassNotFoundException: jakarta.servlet.http.HttpSessionBindingListener
```

**Root cause:** Starting from `vaadin-testbench-core:10.1.0-rc1` (Vaadin 25.1.x), the `vaadin-testbench-unit` dependency was dropped, which previously provided `jakarta.servlet:jakarta.servlet-api` transitively. This causes `SerializationTest` to fail at runtime.

**Fix:** Add an explicit `jakarta.servlet:jakarta.servlet-api:6.1.0:test` dependency to the `v25` profile in `pom.xml`:

```xml
<dependency>
    <groupId>jakarta.servlet</groupId>
    <artifactId>jakarta.servlet-api</artifactId>
    <version>6.1.0</version>
    <scope>test</scope>
</dependency>
```

Reported via mstahv/vaadin-ecosystem-build#105.

--- a/pom.xml
+++ b/pom.xml
@@ -475,6 +475,12 @@
 					<artifactId>vaadin-dev</artifactId>
 					<optional>true</optional>
 				</dependency>
+				<dependency>
+					<groupId>jakarta.servlet</groupId>
+					<artifactId>jakarta.servlet-api</artifactId>
+					<version>6.1.0</version>
+					<scope>test</scope>
+				</dependency>
 			</dependencies>
 		</profile>
         	<profile>

